spark/src/main/kotlin/com/adevinta/spark/SparkTheme.kt

```diff
@@ -0,0 +1,244 @@
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.adevinta.spark
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CutCornerShape
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ProvideTextStyle
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.movableContentOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontFamily.Companion
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.adevinta.spark.tokens.LocalSparkColors
+import com.adevinta.spark.tokens.LocalSparkShapes
+import com.adevinta.spark.tokens.LocalSparkTypography
+import com.adevinta.spark.tokens.SparkColors
+import com.adevinta.spark.tokens.SparkShapes
+import com.adevinta.spark.tokens.SparkTypography
+import com.adevinta.spark.tokens.asMaterial3Colors
+import com.adevinta.spark.tokens.asMaterial3Shapes
+import com.adevinta.spark.tokens.asMaterial3Typography
+import com.adevinta.spark.tokens.darkSparkColors
+import com.adevinta.spark.tokens.debugColors
+import com.adevinta.spark.tokens.lightSparkColors
+import com.adevinta.spark.tokens.sparkShapes
+import com.adevinta.spark.tokens.sparkTypography
+import com.adevinta.spark.tokens.updateColorsFrom
+import com.adevinta.spark.tools.preview.ThemeVariant
+import com.adevinta.spark.tools.preview.UserType
+
+@Composable
+fun SparkTheme(
+    // We don't want to automatically support dark theme in the app but still want it in the previews
+    colors: SparkColors = SparkTheme.colors,
+    shapes: SparkShapes = SparkTheme.shapes,
+    typography: SparkTypography = SparkTheme.typography,
+    useSparkTokensHighlighter: Boolean = false,
```

Comment:
I made them arguments so that we can easily configure them with an FF or a RC without a strong dependency to LBC